src/etc/one-context.d/loc-10-network##apk.one

```diff
@@ -163,7 +163,7 @@ EOT
 
 get_interface_mac()
 {
-    ip link show | awk '/^[0-9]+: [A-Za-z0-9]+:/ { device=$2; gsub(/:/, "",device)} /link\/ether/ { print device " " $2 }'
+    ip link show | awk '/^[0-9]+: [A-Za-z0-9@]+:/ { device=$2; gsub(/:/, "",device); split(device,dev,"\@")} /link\/ether/ { print dev[1]  " " $2 }'
```

Comment:
This warning `awk: cmd. line:1: warning: escape sequence `\@' treated as plain `@'` must be fixed. The same for the rest `ip link`commands.